### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by alanpoulain**

### CODE REVIEW

File: achievements/api-platform.md [hunk 1]
```diff
@@ -1,51 +1,48 @@
-# API Platform Project achievements
+# API Platform Project Achievements
```
Comment (jfhovinne) [inline achievements/api-platform.md]: I think we use 'achievements' in the other pages.

File: achievements/api-platform.md [hunk 2]
```diff
@@ -1,51 +1,48 @@
...
 
-Since 2 years from now, API Platform supports subresources, which allows to access a relation from another.
-It is used a lot and is very powerful, but since it has a lot of complexity and is limited.
+Since 2 years ago, API Platform supports subresources, which allows to access a relation from a resource.
+It is used a lot and is very powerful, but it has a lot of complexity and is limited.
 
-With the help of @vincentchalamon, @soyuka & @torreytsui, we made some research on how to improve the subresources. How to reduce the complexity of the implementation, the management of collection is very important for us.
+With the help of [@vincentchalamon](https://github.com/vincentchalamon), [@soyuka](https://github.com/soyuka) and [@torreytsui](https://github.com/torreytsui), some research was made on how to improve the subresources by reducing the complexity of the implementation and by enabling the POST/PUT operations for the subresources (https://github.com/api-platform/core/pull/2598, https://github.com/api-platform/core/pull/2428).
```
Comment (jfhovinne) [inline achievements/api-platform.md]: can you make update the last links according to #17 ?

File: achievements/api-platform.md [hunk 3]
```diff
@@ -1,51 +1,48 @@
...
 
-Thanks to @jewome62, it's now possible to populate an object recursively with the serializer, to normalize a DateTimeZone object, and to get an instanciated object from an associative array of values.
+Thanks to [@jewome62](https://github.com/jewome62), it's now possible to populate an object recursively with the serializer to normalize a `DateTimeZone` object, and to get an instanciated object from an associative array of values.
 
-@soyuka did a lot to close stale issues, along with @vincentchalamon.
+[@soyuka](https://github.com/soyuka) did a lot to close stale issues, along with [@vincentchalamon](https://github.com/vincentchalamon).
 
-@gregoirehebert fixed the fact the cache was still purged even if there was no GET operation on an ApiResource, and added a way to switch version for the documentation.
+[@gregoirehebert](https://github.com/gregoirehebert) fixed an issue where the cache was still purged even if there was no GET operation on an ApiResource, and added a way to switch the API Platform version for the documentation (https://github.com/api-platform/website/pull/103).
```
Comment (jfhovinne) [inline achievements/api-platform.md]: same remark for the link here

File: achievements/api-platform.md [hunk 4]
```diff
@@ -1,51 +1,48 @@
...
 
-@Deuchnord for his part worked on the client generator, you'll be able soon to use Mercure on the Vue.js generated client.
+[@Deuchnord](https://github.com/Deuchnord) worked on the client generator, you'll be able to use Mercure on the Vue.js generated client soon.
 
-@alanpoulain worked on multiple things about graphQL, documentation and mongodb, and the event system of API Platform:
- 	
-- Relay Input Object Mutations Specification was not fully respected: https://facebook.github.io/relay/graphql/mutations.htm#sec-Mutation-inputs
+[@alanpoulain](https://github.com/alanpoulain) worked on multiple things about GraphQL, MongoDB, and the event system of API Platform:
+
+- The event system is being reworked to have internal events in API Platform instead of being based on the Symfony ones. It will allow to be decoupled from Symfony and to add events for GraphQL (https://github.com/api-platform/core/pull/2506)
```
Comment (jfhovinne) [inline achievements/api-platform.md]: here also

File: achievements/api-platform.md [hunk 5]
```diff
@@ -1,51 +1,48 @@
...
+- Relay Input Object Mutations Specification (GraphQL) was not fully respected (https://github.com/api-platform/core/pull/2696)
```
Comment (jfhovinne) [inline achievements/api-platform.md]: and here

**Review by jfhovinne — CHANGES_REQUESTED** [review]
Nice improvements, thanks! Minor fixes needed.

File: achievements/api-platform.md (hunk 1 above)
Comment (alanpoulain) [reply achievements/api-platform.md]: I think this is better like this (https://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html), so I will do the modification for the other pages as well.

**Comment by alanpoulain** [comment]
Done.

File: achievements/api-platform.md (hunk 1 above)
Comment (jfhovinne) [reply achievements/api-platform.md]: I do not have a strong opinion on this, so would use the EC style guide as a reference: https://ec.europa.eu/info/sites/info/files/whats_new_style_guide_en.pdf
In 4.10, I see that titles take a capital only on the first word.

**Comment by alanpoulain** [comment]
Capital letters reverted.

**Review by jfhovinne — APPROVED** [review]

**Comment by jfhovinne** [comment]
Thanks !

**Pull request merged**